src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -74,11 +74,7 @@
 import org.jabref.gui.help.ErrorConsoleAction;
 import org.jabref.gui.help.HelpAction;
 import org.jabref.gui.help.SearchForUpdateAction;
-import org.jabref.gui.importer.ImportCommand;
-import org.jabref.gui.importer.ImportEntriesDialog;
-import org.jabref.gui.importer.ManageCustomImportsAction;
-import org.jabref.gui.importer.NewDatabaseAction;
-import org.jabref.gui.importer.NewEntryAction;
+import org.jabref.gui.importer.*;
```

Comment:
Please adhere our Java style (see https://github.com/JabRef/jabref/wiki/Guidelines-for-setting-up-a-local-workspace)